stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt

```diff
@@ -534,17 +534,10 @@ public class MessageListView : ConstraintLayout {
                 }
             }
 
-            binding.scrollToBottomButton.setScrollButtonViewStyle(requireStyle().scrollButtonViewStyle)
-            scrollHelper.scrollToBottomButtonEnabled = requireStyle().scrollButtonViewStyle.scrollButtonEnabled
-
-            NewMessagesBehaviour.parseValue(
-                tArray.getInt(
-                    R.styleable.MessageListView_streamUiNewMessagesBehaviour,
-                    NewMessagesBehaviour.COUNT_UPDATE.value
-                )
-            ).also {
-                scrollHelper.alwaysScrollToBottom = it == NewMessagesBehaviour.SCROLL_TO_BOTTOM
-            }
+            val style = requireStyle()
+            binding.scrollToBottomButton.setScrollButtonViewStyle(style.scrollButtonViewStyle)
```

Comment:
This works and it's fine parsing the scroll behavior, but can we make the default scroll behavior scroll to bottom only when it's "our" message?

We shouldn't be scrolling to bottom by default, for all messages - we should keep the scroll state if someone else sends a message and we scrolled away.

Reply:
The behaviour that you described is the `COUNT_UPDATE` which counts new messages for other users and scrolls when the current user send a new message. That's the default behaviour for the SDK, but in the sample app this is set to `SCROLL_TO_BOTTOM`. We can change it to `COUNT_UPDATE` to the sample app, what do you think?

Reply:
Sorry for the late reply - yeah would be good to have that as the default for the sample too, so that we can test it against more common use cases!